### PR TITLE
Fix code smells suggested by sonarqube part - 9

### PR DESCRIPTION
**Pull request by anandjakhaniya**
## Impact and Context

**After changes:**
image

## Testing

- [ ] Deployed the service to dev env
- [ ] Used functionality on dev env 
- [ ] Added unit test(s)
- [ ] Added integration test(s)

### CODE REVIEW

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonHibernateUtil.java [hunk 1]
```diff
@@ -61,95 +61,94 @@ public Connection getConnection() throws SQLException {
...
-        LOGGER.trace("connectionString {}", connectionString);
-        // Create registry builder
-        StandardServiceRegistryBuilder registryBuilder =
-            new StandardServiceRegistryBuilder().applySettings(configuration.getProperties());
-        registry = registryBuilder.build();
-        var metaDataSrc = new MetadataSources(registry);
-        for (Class<?> entity : entities) {
-          metaDataSrc.addAnnotatedClass(entity);
-        }
+      return validateConnectionAndFetchExistingSessionFactory(sessionFactory);
```
Comment (ghost) [inline backend/common/src/main/java/ai/verta/modeldb/common/CommonHibernateUtil.java:64]: why did you move this under the `if`? It currently looks like the logic has been inverted since it went from the `else` to the `if`

File: backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java [hunk 2]
```diff
@@ -118,18 +109,12 @@ private void deleteProjects(Session session) {
...
         var updateDeletedStatusExperimentQueryString =
-            new StringBuilder("UPDATE ")
-                .append(ExperimentEntity.class.getSimpleName())
-                .append(" exp ")
-                .append("SET exp.")
-                .append(ModelDBConstants.DELETED)
-                .append(" = :deleted ")
-                .append(" WHERE exp.")
+            new StringBuilder("UPDATE ExperimentEntity exp SET exp.deleted = :deleted WHERE exp.")
                 .append(ModelDBConstants.PROJECT_ID)
```
Comment (ghost) [inline backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java]: Finish converting this to a string instead of a string builder... like
`"UPDATE ExperimentEntity exp SET exp.deleted = :deleted WHERE exp." + ModelDBConstants.PROJECT_ID + " IN (:projectIds)"` instead of a combination of string builder and strings.

File: backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java [hunk 3]
```diff
@@ -189,19 +167,14 @@ private void deleteExperiments(Session session) {
...
         var transaction = session.beginTransaction();
         var updateDeletedStatusExperimentRunQueryString =
-            new StringBuilder("UPDATE ")
-                .append(ExperimentRunEntity.class.getSimpleName())
-                .append(" expr ")
-                .append("SET expr.")
-                .append(ModelDBConstants.DELETED)
-                .append(" = :deleted ")
-                .append(" WHERE expr.")
+            new StringBuilder(
```
Comment (ghost) [inline backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java]: Same here; finish converting fully to a string as opposed to a combo of StringBuilder and String

File: backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java [hunk 4]
```diff
@@ -374,19 +329,13 @@ private void deleteDatasets(Session session) {
...
         var transaction = session.beginTransaction();
         var updateDeletedStatusDatasetVersionQueryString =
-            new StringBuilder("UPDATE ")
-                .append(DatasetVersionEntity.class.getSimpleName())
-                .append(" dv ")
-                .append("SET dv.")
-                .append(ModelDBConstants.DELETED)
-                .append(" = :deleted ")
-                .append(" WHERE dv.")
+            new StringBuilder("UPDATE DatasetVersionEntity dv SET dv.deleted = :deleted WHERE dv.")
```
Comment (ghost) [inline backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java]: same as above

File: backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java [hunk 5]
```diff
@@ -488,10 +415,10 @@ private void deleteRepositories(Session session) {
           var deleteTagsHql =
               new StringBuilder("DELETE " + TagsEntity.class.getSimpleName() + " te where te.id.")
                   .append(ModelDBConstants.REPOSITORY_ID)
```
Comment (ghost) [inline backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java]: You should make this all a string (like I mentioned above); then you won't need the `REPO_ID_QUERY_POST_APPENDER` variable at all.

File: backend/src/test/java/ai/verta/modeldb/RepositoryTest.java [hunk 6]
```diff
@@ -196,20 +201,16 @@ public void createDeleteRepositoryNegativeTest() {
...
+                                .setWorkspaceName(testUser1UserName)
+                                .setName(repo1)
                                 .build())
                         .build())
                 .setRepository(Repository.newBuilder().setName(repo1))
                 .build();
         versioningServiceBlockingStub.createRepository(setRepository);
         Assert.fail();
       } catch (StatusRuntimeException e) {
-        if (testConfig.hasAuth()) {
```
Comment (ghost) [inline backend/src/test/java/ai/verta/modeldb/RepositoryTest.java]: why did you get rid of this check?

File: backend/src/test/java/ai/verta/modeldb/TestSequenceSuite.java [hunk 7]
```diff
@@ -28,10 +25,10 @@
   RepositoryTest.class,
   CommitTest.class,
   MetadataTest.class,
-  DiffTest.class,
-  BlobEquality.class,
-  BlobProtoEquality.class,
-  DiffAndMerge.class,
+  //   DiffTest.class,
```
Comment (ghost) [inline backend/src/test/java/ai/verta/modeldb/TestSequenceSuite.java]: Do these tests not exist anymore? Why are they commented out?

File: backend/common/src/main/java/ai/verta/modeldb/common/CommonHibernateUtil.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/common/src/main/java/ai/verta/modeldb/common/CommonHibernateUtil.java:64]: it should have like `if (sessionFactory != null) {` but i forget to change that condition so its a bug. Changing..

File: backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java (hunk 2 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java]: Updated to string instead of string builder

File: backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java (hunk 3 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java]: Updated to string instead of string builder

File: backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java (hunk 4 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java]: Updated to string instead of string builder

File: backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java (hunk 5 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/cron_jobs/DeleteEntitiesCron.java]: Updated to string instead of string builder and removed `REPO_ID_QUERY_POST_APPENDER`

File: backend/src/test/java/ai/verta/modeldb/RepositoryTest.java (hunk 6 above)
Comment (anandjakhaniya) [reply backend/src/test/java/ai/verta/modeldb/RepositoryTest.java]: Test class committed by mistake so removed from this PR

File: backend/src/test/java/ai/verta/modeldb/TestSequenceSuite.java (hunk 7 above)
Comment (anandjakhaniya) [reply backend/src/test/java/ai/verta/modeldb/TestSequenceSuite.java]: Test class committed by mistake so removed from this PR

**Review by ghost — APPROVED** [review]
